catalog/spatial/kml/spatial-kml-transformer/src/test/java/org/codice/ddf/spatial/kml/converter/TestKmlToMetacard.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.spatial.kml.converter;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalToIgnoringWhiteSpace;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+
+import java.io.InputStream;
+
+import org.junit.Test;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.MetacardImpl;
+import de.micromata.opengis.kml.v_2_2_0.Kml;
+import de.micromata.opengis.kml.v_2_2_0.MultiGeometry;
+import de.micromata.opengis.kml.v_2_2_0.Placemark;
+
+public class TestKmlToMetacard {
+
+    @Test
+    public void testConvertKmlMultiGeometryToMetacardWithBbox() {
+        InputStream stream =
+                TestKmlToJtsConverter.class.getResourceAsStream("/kmlMultiGeometry.kml");
+
+        Kml kml = Kml.unmarshal(stream);
+
+        assertThat(kml, notNullValue());
+
+        Metacard metacard = KmlToMetacard.from(new MetacardImpl(), kml);
+        assertThat(metacard, notNullValue());
+
+        MultiGeometry kmlMultiGeometry =
+                ((MultiGeometry) ((Placemark) kml.getFeature()).getGeometry());
+        assertThat(kmlMultiGeometry, notNullValue());
+
+        com.vividsolutions.jts.geom.Geometry jtsGeometryCollectionGeometry =
```

Comment:
Drop `com.vividsolutions.jts.geom.` in this class?

Reply:
Dropped